internal/turso/databases.go

```diff
@@ -227,8 +237,11 @@ func (d *DatabasesClient) Usage(database string) (DbUsage, error) {
 	}
 	defer r.Body.Close()
 
```

Comment:
While you are at it, could you add a check here for 200 status code and return an error otherwise?